pyuvdata/calfits.py

```diff
@@ -257,6 +275,7 @@ def write_calfits(self, filename, run_check=True,
 
         prihdr['CTYPE3'] = ('TIME', 'Time axis.')
         prihdr['CUNIT3'] = ('JD', 'Time in julian date format')
+        prihdr['CRPIX3'] = 1
         prihdr['CRVAL3'] = self.time_array[0]
```

Comment:
It looks like there is a check on the time axis and it would fail if the times weren't in order because it checks that the np.diff of the time array is constant.